bugfix: detective's access to the shuttle

syndicatecat opened the pull request:
## Описание

Позволяет детективу открывать шлюзы к шаттлу в отбытии и шлюзы отсека службы безопасности на шаттле

## Ссылка на предложение/Причина создания ПР

bugfix

syndicatecat committed:
fix

Daeberdir reviewed (changes requested):
По пути `_maps\map_files\shuttles\...` стоит всем файлам с `emergency_`  в начале провести аналогичные изменения.

syndicatecat committed:
add

zuhariii commented:
> По пути `_maps\map_files\shuttles\...` стоит всем файлам с `emergency_` в начале провести аналогичные изменения.

готово

syndicatecat committed:
Merge branch 'master220' into shuttlesecdetective

Daeberdir committed:
Brigadier's tweaks.

- Аналогичное расширение твика на остальные станционные мапы.
- На Фаррагусе под входом в отсек СБ в отбытии вернул тайл пола на место.

Daeberdir reviewed (approved):
Туда должен иметь доступ как детектив, так и бригмед.
